web_app/src/form.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@font-face {
+	font-family: 'Raleway', sans-serif;
+}
+
+.modal {
+	font-family: 'Raleway';
+	display: none;
+	position: fixed;
+	z-index: 1;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgba(0,0,0,0.4);
+}
+.modal-content {
```

Comment:
If I'm understanding the code right, let's rename this to `.modal` and rename the other one to "background" or something like that. We also might want to give this a fixed pixel width (not height), it might make removing the `<div>` that's surrounding inputs a bit easier